web2.0/src/Components/Historic/HistData.js

```diff
@@ -0,0 +1,110 @@
+import { useState, useEffect } from 'react';
+import WeatherCard from '../weather-card/WeatherCard';
+
+const NUM_DATA_POINTS = 12;
+
+const HistData = () => {
+  const [currentData, setCurrentData] = useState();
+
+  const getData = async () => {
+    const windResponse = await fetch('http://localhost:8000/wind');
+    const windData = await windResponse.json();
+    const swellResponse = await fetch('http://localhost:8000/swell');
+    const swellData = await swellResponse.json();
+    setCurrentData({ windData, swellData });
+  };
+
+  useEffect(() => {
+    getData();
+  }, []);
+
+  const makeHistData = () => {
+    const histData = [];
+
+    for (let i = 0; i < NUM_DATA_POINTS; i++) {
+      histData.push(
+        <div key={i} className='hist-data-container'>
+          <WeatherCard
+            weatherDataLabel={''}
+            weatherData={
+              currentData?.windData[i].time.hour + ':' + currentData?.windData[i].time.min
```

Comment:
I recommend creating a helper function to create the formatted time object - similar to what you did in utils.

Even if this logic was not to be reused, I find that helper functions can help with the maintainability and readability of your code, as the names of the function themselves convey a lot of meaning.

Let me know if you have any questions.